cmd/secret/cmd/root.go

```diff
@@ -0,0 +1,83 @@
+// Package cmd provides functions set and get with cobra library
+package cmd
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/go-itools-internship/go-secret/pkg/crypto"
+	"github.com/go-itools-internship/go-secret/pkg/io/storage"
+	"github.com/go-itools-internship/go-secret/pkg/provider"
+	"github.com/spf13/cobra"
+)
+
+type root struct {
+	key       *string
+	cipherKey *string
+	value     *string
+	path      *string
+	cmd       *cobra.Command
+}
+
+func (r *root) Execute(ctx context.Context) error {
+	return r.cmd.ExecuteContext(ctx)
+}
+
+func New(version string) *root {
+	var secret = &cobra.Command{
+		Use:     "secret",
+		Short:   "Root includes cli commands",
```

Comment:
Description should give a hint for a user what this tool is doing. Still not clear.